README.md

```diff
@@ -1,7 +1,7 @@
 # learn-maintenance
 
 ### Project Status
-[![Coverage Status](https://coveralls.io/repos/github/bquan0/learn-maintenance/badge.svg?branch=badges)](https://coveralls.io/github/bquan0/learn-maintenance?branch=badges)
+[![Coverage Status](https://coveralls.io/repos/github/CNERG/learn-maintenance/badge.svg?branch=main)](https://coveralls.io/github/CNERG/learn-maintenance?branch=main)
```

Comment:
I've just changed the Github org name to be lower case `cnerg`, but I have no idea how this will be reflected on coveralls???

Reply:
It seems to me that the change in the organization name didn't affect anything on coveralls. I tried adding some repos from CNERG to my account on coveralls.io and when I clicked the github link of the learn-maintenance repo it took me to `cnerg/learn-maintenance `

Reply:
Nevermind, the change in the organization name did have an effect on coveralls. I was running some workflows in #17 and the workflows couldn't find the coveralls repository. I fixed it by deleting the repository on coveralls and adding it back (though this deleted all the data that we previously had)